packages/edit-site/src/components/global-styles/block-preview-panel.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { BlockPreview } from '@wordpress/block-editor';
+import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
+import { useResizeObserver } from '@wordpress/compose';
+
+// const normalizedWidth = 248;
+const normalizedHeight = 152;
+
+const BlockPreviewPanel = ( { name } ) => {
+	const blockExample = getBlockType( name )?.example;
+	const panelStyles = {
+		width: '100%',
+		height: `${ normalizedHeight }px`,
+		overflow: 'auto',
+	};
+	const [ containerResizeListener, { width: containerWidth } ] =
+		useResizeObserver();
+
+	return ! blockExample ? null : (
+		<div
+			className="edit-site-block-styles-preview__panel"
+			style={ panelStyles }
+		>
+			<div style={ { position: 'relative', width: '100%', height: 0 } }>
```

Comment:
What is this container for? It's odd to have a div with 0 height. Could `containerResizeListener` live directly in `.edit-site-block-styles-preview__panel`?

Reply:
removed the container 👍